readme.html

```diff
@@ -50,7 +50,7 @@ <h3>ReadMe for Unicode <abbr title="Common Locale Data Repository">CLDR</abbr> v
     <p>Licenses:</p>
     <ul>
  	  <li>Usage of CLDR data and software is governed by the <a href="http://www.unicode.org/copyright.html">Unicode Terms of Use</a>,
- 	    a copy of which is included as unicode-license.txt.</li>
+ 	    a copy of which is included as LICENSE.</li>
```

Comment:
shouldn't this also be a link?

Reply:
could be good.  The whole readme should also go to .md